requirements_dev.txt

```diff
@@ -272,58 +255,53 @@ pytz==2023.2
 pyyaml==6.0
     # via
     #   -r ./requirements.txt
+    #   drf-yasg
     #   tablib
-pyzmq==25.0.2
+pyzmq==25.1.0
     # via locust
-requests==2.28.2
+requests==2.31.0
     # via
     #   -r ./requirements.txt
-    #   coreapi
     #   datapunt-authorization-django
     #   locust
 roundrobin==0.0.4
     # via locust
-ruamel-yaml==0.17.21
-    # via
-    #   -r ./requirements.txt
-    #   drf-yasg
-ruamel-yaml-clib==0.2.7
-    # via
-    #   -r ./requirements.txt
-    #   ruamel-yaml
```

Comment:
Zijn deze dependencies voor ons obsolete geworden? Met andere woorden, gebruiken we deze niet meer?